components/SecondHeading/index.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const SecondHeading = styled.h2`
+  text-align: left;
+  margin: 1rem;
+  padding: 1rem;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  background-color: white;
```

Comment:
You use white as backgroundcolor in other components as well. I would recommend to crrate a variable in Global Styles and use it in the components. That way you can easily switch the background-color if needed e.g. for a Dark Mode